subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -512,17 +516,51 @@ void ull_sync_iso_setup(struct ll_sync_iso_set *sync_iso,
 
 	interval_us -= lll->window_widening_periodic_us;
 
+	/* Calculate ISO Receiver BIG event timings */
+	pdu_spacing = PDU_BIS_US(lll->max_pdu, lll->enc, lll->phy,
+				 PHY_FLAGS_S8) +
+		      EVENT_MSS_US;
+	ctrl_spacing = PDU_BIS_US(sizeof(struct pdu_big_ctrl), lll->enc,
+				  lll->phy, PHY_FLAGS_S8);
+
+	/* Number of maximum BISes to sync from the first BIS to sync */
+	/* NOTE: When ULL scheduling is implemented for subevents, then update
+	 * the time reservation as required.
+	 */
+	num_bis = lll->num_bis - (stream->bis_index - 1U);
+
+	if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_ISO_RESERVE_MAX)) {
+		/* Maximum time reservation for both sequential and interleaved
+		 * packing.
+		 */
+		slot_us = (pdu_spacing * lll->nse * num_bis) + ctrl_spacing;
+
+	} else if (lll->bis_spacing >= (lll->sub_interval * lll->nse)) {
+		/* Time reservation omitting PTC subevents in sequetial
+		 * packing.
+		 */
+		slot_us = pdu_spacing * ((lll->nse * num_bis) - lll->ptc);
+
+	} else {
+		/* Time reservation omitting PTC subevents in interleaved
+		 * packing.
+		 */
+		slot_us = pdu_spacing * ((lll->nse - lll->ptc) * num_bis);
```

Comment:
Are there any risk of these to overflow? 
You are effectively multiplying a `uint32_t` with another value and storing the result in a `uint32_t`.

Reply:
Added comment:

https://github.com/zephyrproject-rtos/zephyr/pull/67533/files#diff-8bb2391bcdaef2f08bcec9ddc367e973742378cf9d051bcd49645c197e6e66d7R532-R541